cmd/juju/cloud/updatecredential.go

```diff
@@ -17,31 +17,31 @@ import (
 )
 
 var usageUpdateCredentialSummary = `
-Updates a controller credential for a cloud.`[1:]
+Updates a remote credential.`[1:]
```

Comment:
I don't know that remote is the best term here. Is remote jaas? Is it another machine?